### ISSUE TITLE
Support running gitstats in a python3-only environment

### ISSUE THREAD

**Issue by xsgordon** [hoxu/gitstats#94]
It's been brought to my attention that Fedora 31 will be going Python3 only:

https://fedoraproject.org/wiki/Changes/F31_Mass_Python_2_Package_Removal
https://bugzilla.redhat.com/show_bug.cgi?id=1737997

As it stands this will result in the removal of gitstats from the Fedora repositories, though I am investigating how to avoid this (by e.g. carrying patches to make it python3 compatible).

I thought I'd ask however if there was any plan with regards to supporting python3 for gitstats, as while Fedora is likely pushing the curve here other distributions may follow suit in future? As far as I can tell this last came up back in 2015 as distros started defaulting /usr/bin/python to python3:

https://github.com/hoxu/gitstats/pull/52
https://bugzilla.redhat.com/show_bug.cgi?id=1583357

**Comment by sebix** [comment]
https://github.com/vifactor/repostat

**Comment by xsgordon** [comment]
Yes, I'm actually aware of repostat. Before I go that route as the package maintainer for gitstats in Fedora I'm checking what the approach of the gitstats project itself is. If in the end the only answer is repostat that's fine but given the change in upstream provenance I likely have to submit it as if it's a new package (or just orphan gitstats and leave someone else to take this up).

**Comment by sebix** [comment]
That repo is dead for years and there are lots of duplicate questions/comments in the issues and pull requests.

For the python3 part see https://github.com/hoxu/gitstats/issues/41 so for this problem only you have a solution. But I'd rather go the way to obsolete this one and use the fork.
